style.css

```diff
@@ -261,3 +272,60 @@ html {
 hr {
   background: #dfe1e6;
 }
+
+.contact {
+  background-color: #6070ff;
+  display: flex;
+  border-top-left-radius: 73px;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  background-image: url(images/contactbg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.contact h2 {
+  font-weight: 700;
+  font-size: 38px;
+  line-height: 52px;
+  color: #fff;
+  margin-top: 74px;
+  padding: 11px;
+}
```

Comment:
- [ ] Please know that the title needs to have more `margin-top` so the text doesn't get mixed with the background, please follow the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to do that, also please know that it's better to use `svg` than `png`, because when using png you'll assure that the quality of the picture won't be bad no matter what size it is. 😁

![image](https://user-images.githubusercontent.com/20567503/177350466-d8019b2a-5245-495a-bee0-6b7e6e7f6f1d.png)

Right now your design looks like this (you can see that the text is mixed with the background and that the quality of the background is a bit bad):

![image](https://user-images.githubusercontent.com/20567503/177350701-bb60a9b5-7642-4911-a4db-6e98172006ff.png)